### DISCUSSION TITLE
Rename Ungoogled Chromium

### DISCUSSION THREAD

**Discussion by ghost** [Eloston/ungoogled-chromium#1307] [Ideas]
There were few issues posted before about this, but no official decision has ever been made it seems. So perhaps we could discuss this further on this "discussions" forum now? If I missed something, maybe Eloston could answer us if he decided to just stick to "ungoogled-chromium". For me it is very weird name, project such as this should have its own good name. Maybe we could all vote on the name? [3]

I wonder why no one though of name like **Phoenix** [4] or play on word **Phynix**.

"a phoenix obtains new life by arising from the ashes of its predecessor"

sounds about right :)

[1] https://github.com/Eloston/ungoogled-chromium/issues/642
[2] https://github.com/Eloston/ungoogled-chromium/issues/738
[3] https://www.quiz-maker.com/results2326949x509E4a43-67
[4] https://en.wikipedia.org/wiki/Phoenix_(mythology)

**Comment by Eloston** [comment]
Yeah, essentially I think ungoogled-chromium is good enough. There's enough material out there to take a stance either way, but really I don't think renaming is going to make a large positive impact on this project. My whole reason for renaming is to have it benefit/please my user base, but I'm not convinced we'd reach any consensus based on what I've seen.

P.S. I still don't think voting will work. At some point in #738, I discovered that anonymity and a trustworthy voting system are very difficult requirements to implement. You can dig around and find my specific comment(s) about it, but I don't think it's worth the time to attempt it.

**Comment by jpsouzamatos** [comment]
Ungoogled derive from the word google, and google is copyrighted by the company google so it's wise to rename the browser as soon as possible in the next release to prevent they using suing as a form of persecution (they'll surely do that if the browser get good market share in the future).
PS: Phoenix is lot cooler name.
